docs/api-proposal/proposed-routes.md

```diff
@@ -0,0 +1,76 @@
+This doc proposes a new set of API endpoints for the multi-jurisdiction audit flow.
+
+Some principles I tried to follow:
+
+- Reuse existing routes when available
+- Use REST conventions
+- Avoid reusing routes across roles. Instead of reusing one route and returning different scoped responses, make different roles query different endpoints with explicit scoping in the route. That way we just have to check if a user has access to the data they are asking for, as opposed to trying to filter data based on their access. (I learned this one the hard way.)
+
+New routes are in **bold**.
+
+# Audit Admin: Audit setup
+
+![](./create-audit.png)
+
+- POST /election/new - create new audit
+- **GET /election** - get all audits
+
+![](upload-jurisdiction-file.png)
+
+- POST /election/<election_id>/jurisdiction/csv - upload CSV file of jurisdictions with JAs
+- GET /election/<election_id>/jurisdiction/csv - download CSV file of jurisdictions with JAs
+
+![](contests.png)
+
+- **PUT /election/<election_id>/contests** - create/update all contests, including contest universe
+- **GET /election/<election_id>/contests** - get all contests
+
+![](audit-settings.png)
+
+- **GET /election/<election_id>** - get audit settings (risk limit, name, etc)
+- **PUT /election/<election_id>** - update audit settings
+
+![](review-and-launch.png)
+
+- **GET /election/<election_id>/round** - check if we have created the first round object yet
```

Comment:
suggest:
`GET /election/<election_id>/round/` to list all the rounds. No special API for the first round, just get all of them.

Reply:
Relying on `/foo` being different from `/foo/` seems likely to bite us down the road. I don’t have a clear example as to why, but… that’s just not how paths work and it feels pretty unintuitive.

Reply:
This is intended to list all rounds. I was just using the singular `round` to match how the rest of the API is named. I would like to eventually move everything to the plural naming convention with no trailing slash.